keypom/keypom-airfoil
# Fix ticket claims

# Description
Fixes the bugs report below:

1
<img width="462" alt="image" src="https://user-images.githubusercontent.com/19740800/221353530-296924d7-5a4d-425e-9311-b70512876420.png">

1. Scanner page show ticket is invalid after it is scanned as valid
Fix: prevent scanner page from claiming when remaining_uses is 1

2. NFT image large in height
<img width="363" alt="image" src="https://user-images.githubusercontent.com/19740800/221353466-d9540e7a-2b59-43d1-8520-36de624b4ce8.png">
Fix: allow the image container to not have fixed height

3. Create wallet in claim pages are not opened in new tab
Fix: When clicking on the wallet link, new tab will be opened for the external wallet, and in 20 seconds, checks will be conducted to see if the drop is claimed.


# How to test
Show steps to replicate and test the feature/fixes in this PR

1. Attempt to claim ticket drop to test the scanner, try to rescan the QR code after is it valid and rescan the QR code when the modal is up
https://ericw-fix-ticket-claims.keypom-airfoil.pages.dev/claim/ticket/v2.keypom.testnet#ed25519:3PcMtrMEf47hz8hxUGJH8L8SMVjTVFsZyb1t71MDRAb2p8BPG5eg2fVASmqYR65X1XkF19icsSeyfQfpjGzFmuYu
https://testnet.keypom-airfoil.pages.dev/scanner

2. Create wallet should open a new tab and check for successful claim in 20s
https://ericw-fix-ticket-claims.keypom-airfoil.pages.dev/claim/nft/v2.keypom.testnet#ed25519:5SoqQWV5ZMjGu1E856EwLY9V6s7zK2oCf8tiWMvywRWPfF6BMcySsdPdKSqynWeAzDdPwMMEdfW5aBCPDSZwfArM

# Screenshots/Screen Recording of Implementation
